colossus-tests/src/test/scala/colossus/core/WorkerItemSpec.scala

```diff
@@ -77,7 +77,7 @@ class WorkerItemSpec extends ColossusSpec {
           }
         }
         val item = new MyItem
-        io ! IOCommand.BindWorkerItem(item)
+        io ! IOCommand.BindWorkerItem(_ => item)
```

Comment:
I think it's kind of overkill, especially since this is more of an internal thing and usually should not be used by users.